core/src/mindustry/core/NetClient.java

```diff
@@ -339,6 +339,13 @@ public static void infoToast(String message, float duration){
 
         ui.showInfoToast(message, duration);
     }
+    
+    @Remote(variants = Variant.both)
+    public static void warningToast(int unicode, String text){
+        if(text == null || mindustry.ui.Fonts.def.getData().getGlyph((char)unicode) == null) return;
```

Comment:
Why use the whole package name?

Reply:
oh i copy pasted it from `Icon` 👍 

`fileTextFillSmall = mindustry.ui.Fonts.getGlyph(mindustry.ui.Fonts.def, (char)61788);`

(but `.def` is fine? shall i import that one, or the `Fonts` class?)

Reply:
Import `Fonts`. `Icon` uses the whole package name because it's generated code.

Reply:
done 👍 

(`Fonts` is already available there without an extra import it seems)